### DISCUSSION TITLE
DDC control is grayed out / Apple Cinema Display X Thunderbolt Display (macbook m1 2023)

### DISCUSSION THREAD

**Discussion by renanmalato** [MonitorControl/MonitorControl#1468] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hi, 

I have an Apple Cinema Display at home and there, with MonitorControl, I could manually put DDC all in 255 and the brightnesses works pretty similar to my macbook pro.

I bought another piece for my office, but it's a Thunderbolt display, I read some articles that they are the same, but at the office I can't put the DDC 255 and the brightnesses is like on 70%. It shows that is 100% but I have the same piece at home and I know there I could set the brightness decent.

At home I did what this guy did on this video
https://youtu.be/SzZHdyRJkws?si=mftO9ByJcUWwH7bS&t=383

But I can't do that at my office.

Someone could please help me?

![screenshot](https://github.com/MonitorControl/MonitorControl/assets/3617070/946d2a1b-ee0c-488a-85b9-2f62786ab9ea)

### Steps to reproduce

1 Monitor control preferences / Display

2 All options are grayed out - except Enable keyboard for control display

### Expected behavior

Use hardware DDC control not grayed out

DDC max able to put 255 

upper the apple cinema display brightnesses

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey 12.5
- Mac model: macbook pro m1  2023
- MonitorControl version: updated today
- Monitor(s): Thunderbolt Display
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Hi, the thunderbolt display is reported to support native brightness control using Apple's APIs, that's why DDC is not enabled. If native control does not actually work, then you can try [BetterDisplay](https://betterdisplay.pro) and use the opion under the display to force it to be detected as a third party display by the app thus enabling DDC - that way you can try whether DDC control actually works with this display or not.
